LEAF_Request_Portal/admin/templates/mod_svcChief.tpl

```diff
@@ -219,15 +222,15 @@ function importUser(serviceID = 0, selectedUserName = '') {
     if (selectedUserName === '') {
         console.log('Invalid username');
     }
+
     $.ajax({
```

Comment:
looks like this needs a fail-catch

Reply:
I changed this to be like others on the page and just return if the data isn't valid.